### DISCUSSION TITLE
Which Repository has the data of the forms

### DISCUSSION THREAD

**Discussion by Ivkrafft** [sulu/SuluFormBundle#370] [Q&A] [answered]
Hey, i cant seem to find the Repository to access the data that has already been submited.

`  
 public function formData( FormRepository $formRepository): Response
    {
        $forms = $formRepository->loadAll();
 `
this is how i want to access the data in my Controller but this repo just has the forms that had been created, not the data thats been submited.

Maybe it's just the entitymanager?

**Comment by alexander-schranz** [comment, marked as answer]
The `Dynamic` Entity contains all the submitted form data: https://github.com/sulu/SuluFormBundle/blob/2.5/Entity/Dynamic.php.

SuluFormBundle already shows you all the submitted data on the Form itself and if configured on the Submitted Page: https://github.com/sulu/SuluFormBundle/blob/2.5/Resources/doc/dynamic.md#list-tab---export

**Reply by Ivkrafft** [reply]
Yes, i found it! The DynamicRepository was what i wanted.
Thank you!
